src/Component/BooksContainer.css

```diff
@@ -2,3 +2,49 @@
   height: 47vh;
   overflow-y: scroll;
 }
+
```

Comment:
- [ ] Please make sure you stick to the given design.

## Your
<img width="400" src="https://user-images.githubusercontent.com/92215878/215270012-f01769a0-b058-4016-b2c5-05e79cbd35a0.png" />

## Zeplin
<img width="400" src="https://user-images.githubusercontent.com/92215878/215269845-8cb71fd1-c9ef-4965-9363-d2ab1ee04e73.png" />